Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -65,17 +68,15 @@ public class OpenApiMetadataPathsFactory
                                     Path = $"{resourcePath}/{{id}}",
                                     PathItem = CreatePathItemForAccessByIdsOperations(r)
                                 }
-                                : null
-#if NETFRAMEWORK
-                            ,
-                            ChangeQueryFeature.IsEnabled && !r.Name.Equals(ChangeQueryFeature.SchoolYearTypesResourceName) && !isCompositeContext
+                                : null,
+                            _apiSettings.Features.Where(s => s.Name == "ChangeQueries").SingleOrDefault().IsEnabled &&
```

Comment:
there is a method on apisettings that does the check for the feature IsFeatureEnabled()

Reply:
Used IsFeatureEnabled method